build/default.conf

```diff
@@ -13,7 +13,22 @@ server {
         try_files $uri =404;
     }
 
+    # Files should be directly accessed
+    location ~* \.[a-zA-Z]+$ {
+        try_files $uri =404;
+    }
+
     location / {
+        # Demo mode params are added to URL when not yet present and DEMO is set to true
+        if ($arg_demo != "true") {
+            set $demo "${DEMO}true";
```

Comment:
Can we make this value a bit more intuitive? I got stuck why we append "true" to "true" ... why not `set $demo "demo:${DEMO}"` and `if ($demo = "demo:true")` for example?